docs/user/science.rst

```diff
@@ -161,116 +161,120 @@ Read the Docs community for science is already big and keeps growing. The :exter
 .. Let's put some logos to sign off
 
 
-How would you use Read the Docs for Science?
---------------------------------------------
+..
+    THE FORM IS DISABLED BECAUSE OF FORM SPAM
```

Comment:
I'm fine if you want to delete the form completely and put a comment like "We used to have a form made with Front here that was remove because of spam. You can recover the code from Git history if required"

Reply:
Because the whole page was already created as a placeholder for a future page, likely on about.readthedocs.org, I feel confident that it will inevitably go away :)